orcid-persistence/src/main/java/org/orcid/persistence/aop/ProfileLastModifiedAspect.java

```diff
@@ -130,7 +127,7 @@ public void updateLastModifiedDateAndIndexingStatus(String orcid) {
         Date last = retrieveLastModifiedDate(orcid);
         LastModifiedMessage mess = new LastModifiedMessage(orcid,last);
         if (messaging.isEnabled()){
-            if(!messaging.send(mess,JmsDestination.UPDATED_ORCIDS)) {
+            if(!messaging.send(mess, JmsDestination.UPDATED_ORCIDS)) {
```

Comment:
@TomDemeranville I modified the code so, here we send messages when someone changes a record (through the UI or the API), and OrcidProfileManagerImpl will be in charge of sending the messages when the record is marked as REINDEX.